cli/lib/exec/run.ts

```diff
@@ -87,7 +88,7 @@ const processRunOptions = (options = {}) => {
       return throwInvalidOptionError(errors.incompatibleHeadlessFlags)
     }
 
-    args.push('--headed', !options.headless)
+    args.push('--headed', String(!options.headless))
```

Comment:
@AtofStryker Is this right to change this to a string? I saw the spec was updated to match this. Just making sure it's intentional. https://github.com/cypress-io/cypress/pull/32063/files#diff-ef1734baae1d58ef930dfc18abd86dc41752c8a1b93a0296ba22c409dad75bb1R64

Reply:
I think the AI did this which I think under the hood is a string but we really should have a 1:1 so I am going to look into reverting it

Reply:
going to resolve